setup.py

```diff
@@ -39,7 +39,7 @@ def run_tests(self):
 
 setup(
     name='vcrpy',
-    version='2.0.1',
+    version='2.0.2',
```

Comment:
I recommend not bumping the build number for a local build... because it opens the opportunity for the upstream fork to release a newer yet lesser version (e.g. `2.0.1.1`). A version of `2.0.1+yg1` is usually best because it signifies "2.0.1" "plus" "yougov's first increment".

Reply:
ok